crates/python/src/qvm/mod.rs

```diff
@@ -16,24 +17,33 @@ mod api;
 
 use api::PyAddressRequest;
 
-wrap_error!(RustQvmError(qcs::qvm::Error));
-py_wrap_error!(api, RustQvmError, QVMError, PyRuntimeError);
-
-py_wrap_type! {
-    PyQvmResultData(QvmResultData) as "QVMResultData"
-}
-
 create_init_submodule! {
-    classes: [PyQvmResultData, PyQvmOptions],
+    classes: [PyQvmResultData, PyQvmOptions, RawQvmReadoutData],
     errors: [QVMError],
     funcs: [py_run, py_run_async],
     submodules: [
         "api": api::init_submodule
     ],
 }
 
+wrap_error!(RustQvmError(qcs::qvm::Error));
```

Comment:
Does moving these after the `create_init_submodule` change anything?

Reply:
I don't think so. Perhaps @Shadow53 moved them for stylistic purposes?